src/View/StringTemplate.php

```diff
@@ -178,10 +178,6 @@ protected function _compileTemplates(array $templates = []): void
         }
         foreach ($templates as $name) {
             $template = $this->get($name);
-            if ($template === null) {
```

Comment:
We should probably throw an exception if a template with the given name is not found.